scapy/plist.py

```diff
@@ -85,6 +93,7 @@ def __repr__(self):
                                ct.punct(">"))
 
     def __getstate__(self):
+        # type: () -> Dict[str, Any]
```

Comment:
You might use a stricter type for `__getstate__ `and `__setstate__` as `res` is `List[PacketField]`, listname is `str` and stats is `List[Packet]`.

Reply:
Made it stricter